aws-iam-role-route53domains-poweruser/main.tf

```diff
@@ -0,0 +1,21 @@
+data "aws_iam_policy_document" "assume-role" {
+  statement {
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::${var.source_account_id}:root"]
+    }
+
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+resource "aws_iam_role" "route53domains-poweruser" {
+  name               = "${var.role_name}"
+  path               = "${var.iam_path}"
+  assume_role_policy = "${data.aws_iam_policy_document.assume-role.json}"
+}
+
+resource "aws_iam_role_policy_attachment" "route53domains-fullaccess" {
+  role       = "${aws_iam_role.ecs-poweruser.name}"
```

Comment:
https://travis-ci.com/chanzuckerberg/cztack/jobs/196016580#L1562 is the test error. This should be `aws_iam_role.route53domains-poweruser.name` (from https://github.com/chanzuckerberg/cztack/pull/94/files#diff-acd550bbd190bc588eb1d9ef19d3b4ceR12)

Reply:
oh whoops - missed one. will change this over